### PR TITLE
refactor: token list api source

### PR DESCRIPTION
**Pull request by stackchain**
### ** Breaking change **
- [x] Updated openswap api to use token-list instead of list
- [x] Updated the types
- [x] Updated manager, helpers and transformers
- [x] Removed volume (which as supply) from the list
- [x] Approved by product

Relates to YOMO-925

---
### Context
The `/list` api is very restricted to the amount of queries 1/hour

### CODE REVIEW

File: apps/wallet-mobile/src/components/AmountItem/AmountItem.tsx [hunk 1]
```diff
@@ -63,13 +62,15 @@ export const AmountItem = ({isPrivacyOff, wallet, style, amount, inWallet, suppl
       </Middle>
 
       <Right>
-        {tokenInfo.kind !== 'nft' && (
+        {tokenInfo.kind !== 'nft' && variant !== 'swap' && (
```
Comment (stackchain) [inline apps/wallet-mobile/src/components/AmountItem/AmountItem.tsx:65]: changed the variant `swap` to don't list supply / pair

File: apps/wallet-mobile/src/features/Swap/common/helpers.ts [hunk 2]
```diff
@@ -73,26 +73,22 @@ function containsOnlyValidChars(str?: string): boolean {
   return typeof str === 'string' && validCharsRegex.test(str)
 }
 
-export const sortTokensByName = (a: Balance.Token, b: Balance.Token, wallet: YoroiWallet) => {
-  const isValidNameA = containsOnlyValidChars(a.info.name)
-  const isValidNameB = containsOnlyValidChars(b.info.name)
-  const isValidTickerA = containsOnlyValidChars(a.info.ticker)
-  const isValidTickerB = containsOnlyValidChars(b.info.ticker)
+export const sortTokensByName = (a: Balance.TokenInfo, b: Balance.TokenInfo, wallet: YoroiWallet) => {
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/Swap/common/helpers.ts:76]: new endpoint returns TokenInfo not Token, adjusted

File: apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/CreateOrder/EditBuyAmount/SelectBuyTokenFromListScreen/SelectBuyTokenFromListScreen.tsx [hunk 3]
```diff
@@ -60,20 +60,20 @@ export const SelectBuyTokenFromListScreen = () => {
 const TokenList = () => {
   const strings = useStrings()
   const wallet = useSelectedWallet()
-  const {pairsByToken} = useSwapTokensByPairToken('')
+  const {onlyVerifiedTokens} = useSwapTokensOnlyVerified()
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/CreateOrder/EditBuyAmount/SelectBuyTokenFromListScreen/SelectBuyTokenFromListScreen.tsx:63]: pre-fetch updated, until we migrate router for tan stack router

File: lerna.json [hunk 4]
```diff
@@ -1,4 +1,9 @@
 {
   "npmClient": "yarn",
-  "version": "independent"
+  "version": "independent",
+  "command": {
+    "run": {
+      "ignore": ["e2e/*"]
```
Comment (stackchain) [inline lerna.json:6]: ignoring e2e on lerna (it was rebuilding it)

File: packages/openswap/src/api.ts [hunk 5]
```diff
@@ -95,13 +96,17 @@ export class OpenSwapApi {
     )
   }
 
-  public async getTokens({policyId = '', assetName = ''} = {}) {
-    const tokens = await getTokens(
+  public async getTokenPairs({policyId = '', assetName = ''} = {}) {
```
Comment (stackchain) [inline packages/openswap/src/api.ts:99]: getTokens is actually getTokenPairs

File: packages/swap/src/helpers/transformers.ts [hunk 6]
```diff
@@ -93,27 +93,11 @@ export const transformersMaker = (
...
-        decimals: info.decimalPlaces,
-        description: info.description,
-        image: info.image,
-        kind: 'ft',
-        symbol: info?.sign,
-        icon: undefined,
-        ticker: info.symbol,
-        metadatas: {},
-      },
+      info: asYoroiBalanceTokenInfo(info),
```
Comment (stackchain) [inline packages/swap/src/helpers/transformers.ts:100]: migrated to the specialized helper

File: apps/wallet-mobile/src/features/Swap/SwapNavigator.tsx [hunk 7]
```diff
@@ -57,7 +57,7 @@ export const SwapTabNavigator = () => {
   }, [aggregatorTokenId, lpTokenHeld, lpTokenHeldChanged])
 
   // pre load swap tokens
-  const {refetch} = useSwapTokensByPairToken('', {suspense: false, enabled: false})
+  const {refetch} = useSwapTokensOnlyVerified({suspense: false, enabled: false})
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/Swap/SwapNavigator.tsx:60]: pre-fetch updated, until we migrate router for tan stack router

**Review by michalszorad — APPROVED** [review]

**Review by banklesss — APPROVED** [review]

**Review by SorinC6 — APPROVED** [review]

**Review by jorbuedo — APPROVED** [review]

**Pull request merged**